design-tokens/tokens/layer1/base.json

```diff
@@ -113,78 +133,150 @@
       "type": "letterSpacing"
     },
     "increased": {
-      "value": "150%",
+      "value": "2%",
       "type": "letterSpacing"
     },
     "decreased": {
-      "value": "-5%",
+      "value": "-2%",
       "type": "letterSpacing"
     }
   },
   "paragraphSpacing": {
-    "h1": {
-      "value": "32",
+    "small": {
+      "value": "1rem",
       "type": "paragraphSpacing"
     },
-    "h2": {
-      "value": "26",
+    "medium": {
+      "value": "1.125rem",
+      "type": "paragraphSpacing"
+    },
+    "large": {
+      "value": "1.375rem",
       "type": "paragraphSpacing"
     }
   },
-  "fontWeights": {
-    "headingRegular": {
-      "value": "Regular",
-      "type": "fontWeights"
-    },
-    "headingBold": {
+  "fontWeight": {
+    "bold": {
       "value": "Bold",
       "type": "fontWeights"
     },
-    "bodyRegular": {
-      "value": "Regular",
+    "semiBold": {
+      "value": "Semi Bold",
       "type": "fontWeights"
     },
-    "bodyBold": {
-      "value": "Bold",
+    "regular": {
+      "value": "Regular",
       "type": "fontWeights"
     }
   },
-  "fontSizes": {
-    "h1": {
-      "value": "{fontSizes.h2} * 1.25",
+  "fontSize": {
+    "headingXXXL": {
+      "value": "2.125rem",
```

Comment:
Current biggest in MIUI is 2.25rem do we want to align?

Reply:
This is based on what a revamp we did to our typescale. The one I shared for review with you @gregaubert.